tests/expectations/metrics/test_core.py

```diff
@@ -294,8 +299,8 @@ def test_column_quoted_name_type_sa(sa):
 
     for column_name in [
         "non_existent_column",
-        '"NAMES"',
-        '"Names"',
+        "?NAMES?",
```

Comment:
@TrangPham I am not familiar with the origin of these markers -- were the question mark and the asterisk mentioned in a specification document (or a ticket)?  Please let me know what use case we are aiming for.  Thank you!

Reply:
Oh I was using these as counter examples as in these should not be valid. 

I just had an idea though. We could take in an optional parameter from the user which is `identifier_delimiter` and then check column names and add that for the user to col string.

Usage would then look like: 
``` 
    col_name = "other name"
    result = validator.expect_column_values_to_be_unique(column=col_name, identifier_delimiter="`")
```